doc/source/parameters/cfd/linear_solver_control.rst

```diff
@@ -189,9 +199,10 @@ Different parameters for the main components of the two geometric multigrid algo
 .. code-block:: text
 
     # General MG parameters
-    set mg verbosity       = quiet
-    set mg min level       = -1
-    set mg level min cells = -1
+    set mg verbosity                   = quiet
+    set mg min level                   = -1
+    set mg level min cells             = -1
+    set mg enable hessians in jacobian = true
```

Comment:
Here the other parameter default value should be specified
set enable hessians in residual = true

Reply:
There is no residual parameter in this case. So, there are three parameters, enable hessians in jacobian, enable hessians in residual and mg enable hessians in jacobian. The first two can enable them for the system operator, while this one only considers the operators of the mg preconditioner.